config.example.yaml

```diff
@@ -34,7 +60,46 @@ users:
 # Members not defined in the group will be removed from it during synchronization.
 groups:
   - name: Test Group
-    email: testgroup@myorganization.com
+    # email of the group; must end with your organization's domain name
+    email: testgroup@myorganization.com 
+    # Permission to contact owner of the group via web UI. Possible values are:
+    # - ALL_IN_DOMAIN_CAN_CONTACT
+    # - ALL_MANAGERS_CAN_CONTACT
+    # - ALL_MEMBERS_CAN_CONTACT
+    # - ANYONE_CAN_CONTACT
+    who_can_contact_owner: ALL_MEMBERS_CAN_CONTACT
+    # Permissions to view group messages. Possible values are:
+    # - ANYONE_CAN_VIEW: Any Internet user 
+    # - ALL_IN_DOMAIN_CAN_VIEW: Anyone in your account
+    # - ALL_MEMBERS_CAN_VIEW: All group members 
+    # - ALL_MANAGERS_CAN_VIEW: Any group manager 
+    who_can_view_members: ALL_MEMBERS_CAN_VIEW
+    # Specifies who can approve members who ask to join groups. Possible values are:
+    # - ALL_MEMBERS_CAN_APPROVE
+    # - ALL_MANAGERS_CAN_APPROVE
+    # - ALL_OWNERS_CAN_APPROVE
+    # - NONE_CAN_APPROVE
+    who_can_approve_members: ALL_MANAGERS_CAN_APPROVE
+    # Permissions to post messages. Possible values are:
+    # - NONE_CAN_POST: The group is disabled and archived; 'is_archived' must be set to true, otherwise will result in an error
+    # - ALL_MANAGERS_CAN_POST: Managers, including group owners
+    # - ALL_MEMBERS_CAN_POST: Any group member
+    # - ALL_OWNERS_CAN_POST: Only group owners 
+    # - ALL_IN_DOMAIN_CAN_POST: Anyone in the account 
+    # - ANYONE_CAN_POST: Any Internet user who can access your Google Groups service 
+    who_can_post: ALL_MEMBERS_CAN_POST
+    # Permission to join group. Possible values are:
+    # - ANYONE_CAN_JOIN: Anyone in the account domain 
+    # - ALL_IN_DOMAIN_CAN_JOIN: Any Internet user who can access your Google Groups service 
+    # - INVITED_CAN_JOIN: Candidates for membership can be invited to join
+    # - CAN_REQUEST_TO_JOIN: Non members can request an invitation to join
+    who_can_join: CAN_REQUEST_TO_JOIN
+    # Identifies whether members external to your organization can join the group. 
+    # Possible values are: "true" or "false"
+    allow_external_members: "false"
```

Comment:
Why are booleans strings in the YAML?

Reply:
Because google API has those fields natively as a string and I have just aligned to it. 
I can add type conversions for those fields!

Reply:
I think that would be good. We (I think) don't follow the GSuite data structures to the letter in our config.yaml, so in this case I would support defining a "sane" YAML and then converting the bool to a string (`fmt.Sprintf("%v", val)`) when talking to the API. :-)

Reply:
I have introduced the conversion with `ParseBool` (and `FormatBool` for other way around), I hope it is okay. 
https://github.com/kubermatic-labs/gman/pull/11/commits/4b47f836cb50d908839f5ffef246927836676bce#diff-65be9b04993ff3a5c2435628e9035455R455